PythonRegression/tests/features/machine1/m1_static_api_tests.feature

```diff
@@ -70,19 +70,19 @@ Feature: Test API calls on Machine 1
 		Given getTrytes is called with the hash static_vals.TEST_HASH
 		Then the response should be equal to static_vals.TEST_TRYTES
 	
-	
-	@transactionsToApprove	
-	Scenario: GetTransactionsToApprove is called
-		Given "getTransactionsToApprove" is called on "nodeA"
-		Then a response with the following is returned: 
-		|keys				|
-		|trunkTransaction	|
-		|branchTransaction	|
-		|duration			|
+#	
+#	@transactionsToApprove	
+#	Scenario: GetTransactionsToApprove is called
+#		Given "getTransactionsToApprove" is called on "nodeA"
+#		Then a response with the following is returned: 
+#		|keys				|
+#		|trunkTransaction	|
+#		|branchTransaction	|
+#		|duration			|
```

Comment:
This commenting was part of the previous commits that contained the older config changes from over a month ago. This test works now, but at the time it was a problem because of the node synchronisation issues we were having with 1.5.0 and 1.5.1. The next PR will contain the commit that un-comments this. Unless you want me to go back through all the commits and cherry pick them more in depth for each PR, this is basically just step 1 of the 3 for the big PR you wanted me to break down.